db/widget_specifications/image_specification.js

```diff
@@ -0,0 +1,53 @@
+const image = {
+	"category": "Dashboard Tools",
```

Comment:
TABS WHAT ARE YOU DOING TO ME WHY DO WE HAVE TABS THE WORLD IS COMING TO END NOTHING IS GOOD IN LIFE 💥 💥 🌎 💥 💥